src/sortStudents.ts

```diff
@@ -1,16 +1,60 @@
 
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[]
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name,
+  Surname,
+  Age,
+  Married,
+  AverageGrade,
```

Comment:
Если задать им значения, то их можно будет использовать в дальнейшем, чтобы сократить количество кода при переборе

```suggestion
  Name = 'name',
  Surname = 'surname',
  Age = 'age',
  Married = 'married',
  AverageGrade = 'grade',
```